tests/app/main/views/test_suppliers.py

```diff
@@ -169,57 +203,59 @@ def teardown_method(self, method):
             (
                 "admin-ccs-data-controller",
                 ["digital-outcomes-and-specialists-3", "g-cloud-10"],
-                ["g-cloud-11", "g-cloud-standstill", "g-cloud-pending", "g-cloud-open"]
+                ["g-cloud-11", "g-cloud-standstill-1", "g-cloud-pending-1", "g-cloud-open-1"]
```

Comment:
Why did you need to append "-1"?

Reply:
There was some function or other that was grabbing the suffix of the framework slug (that's a thought, I wonder what it did for `digital-outcomes-and-specialists`...)

Reply:
It's the `FrameworkStub` - for slugs starting `g-cloud-` it derives the lots and so on.

Reply:
Ah fair enough